configurations/__init__.py

```diff
@@ -32,3 +32,4 @@ def setup(app=None):
     in your docs' ``conf.py``.
     """
     _setup()
+    return {'version': __version__, 'parallel_read_safe': True}
```

Comment:
Yeah, the multiprocessing support is a new feature not a bugfix, so I think we'll be good.